### ISSUE TITLE
Only Runs On Addon Directory

### ISSUE THREAD

**Issue by chadhietala** [pzuraq/ember-compatibility-helpers#29]
This seems to only run on the `addon` directory when used in an addon. When I moved the code that was using ember-compatibilty-helpers to the `addon` directory the inlining of the boolean worked. Prior to that I would get a runtime exception about ember-compatibilty-helpers missing.

**Comment by pzuraq** [comment]
That makes some sense, this addon is a plugin that adds itself to its parent's babel configuration, so maybe it's only running against the `/addon` dir because that's all that `ember-cli-babel` does when included in an Ember addon? e.g., `app` gets merged directly into the root app without any transforms being run on it

**Comment by rwjblue** [comment]
> This seems to only run on the addon directory when used in an addon. When I moved the code that was using ember-compatibilty-helpers to the addon directory the inlining of the boolean worked.

Yep, sounds like things are functioning as intended. This is exactly the reason that we advise folks that **all** addon code should be in the `addon/` directory, and the `app/` directory should _only_ contain re-exports.

**Comment by ppcano** [comment]
I experienced the opposite behavior (not running in the addon directory) and reported the issue at #32.

**Comment by rileyhilliard** [comment]
> > This seems to only run on the addon directory when used in an addon. When I moved the code that was using ember-compatibilty-helpers to the addon directory the inlining of the boolean worked.
> 
> Yep, sounds like things are functioning as intended. This is exactly the reason that we advise folks that **all** addon code should be in the `addon/` directory, and the `app/` directory should _only_ contain re-exports.

What about the `fastboot/` directory? I think I am hitting this issue for code in `fastboot/`:
```
Could not find module `ember-compatibility-helpers` imported from `<PATH TO FILE THAT IMPORTS ember-compatibility-helpers>`
```

**Comment by rwjblue** [comment]
`fastboot` is roughly the same treatment as `app` I think (though we'd have to double check with the implementation in `ember-cli-fastboot`).
